content/01.abstract.md

```diff
@@ -1,16 +1,13 @@
-Unfortunately writing useful label functions requires substantial error analysis and is a nontrivial task: in early efforts to use data programming we found that producing each label function could take a few days.
-Producing a biomedical knowledge base with multiple node and edge types could take hundreds or possibly thousands of label functions.
-In this paper we sought to evaluate the extent to which label functions could be re-used across edge types. 
-We used a subset of Hetionet v1 that centered on disease, compound, and gene nodes to evaluate this approach.
-We compared a baseline distant supervision model with the same distant supervision resources added to edge-type-specific label functions, edge-type-mismatch label functions, and all label functions.
-We confirmed that adding additional edge-type-specific label functions improves performance.
-We also found that adding one or a few edge-type-mismatch label functions nearly always improved performance.
-Adding a large number of edge-type-mismatch label functions produce variable performance that depends on the edge type being predicted and the label function's edge type source.
-Lastly, we show that this approach, even on this subgraph of Hetionet, could add new edges to Hetionet v1 with high confidence.
-We expect that practical use of this strategy would include additional filtering and scoring methods which would further enhance precision.
+Knowledge graphs support multiple research efforts by providing contextual information for biomedical entities, constructing networks, and supporting the interpretation of high-throughput analyses. 
+Hetionet v1 is a knowledge graph derived from various biomedical databases. 
+These databases are populated via some form of manual curation, which is difficult to scale in the context of an increasing publication rate. 
+Data programming is a paradigm that circumvents this arduous process by combining databases with simple rules and heuristics written as label functions, which are programs designed to automatically annotate textual data. 
+Unfortunately, writing a useful label function requires substantial error analysis and is a nontrivial task that takes multiple days per function.
+This makes populating a knowledge graph with multiple nodes and edge types practically infeasible. 
+We sought to accelerate the label function creation process by evaluating the extent to which label functions could be re-used across multiple edge types.
+We used a subset of Hetionet v1 centered on disease, compound, and gene entities to evaluate label function re-use. 
+We determined the best label function combination by comparing a baseline database-only model with the same model but added edge-specific or edge-mismatch label functions.
+We confirmed that adding additional edge-specific rather than edge-mismatch label functions often improves text annotation and show that this approach can incorporate novel edges into Hetionet v1. 
```

Comment:
```suggestion
We confirmed that adding additional edge-specific rather than edge-mismatch label functions often improves text annotation and show that this approach can incorporate novel edges into our source knowledge graph. 
```